deploy-board/deploy_board/templates/deploys/deploy_progress.tmpl

```diff
@@ -1,241 +1,348 @@
+                                    {% for agentStat in report.firstTimeAgentStats|slice:":100" %}
+                                        <a href="/env/{{env.envName}}/{{env.stageName}}/host/{{ agentStat.agent.hostName }}"
+                                           type="button" class="deployToolTip btn btn-xs {{ agentStat | agentButton }} host-btn"
+                                           title="{{ agentStat | agentTip }}">
+                                           {% if report.showMode == "complete" %}
+                                           <small>{{ agentStat.agent.hostName }}</small>
+                                           {% endif %}
+                                           <i class="fa fa-fw {{ agentStat | agentIcon }}"></i>
+                                        </a>
+                                    {% endfor %}
+                                    <i>Results limited to 100</i>
+                                {% else %}
+                                    {% for agentStat in report.firstTimeAgentStats %}
+                                        <a href="/env/{{env.envName}}/{{env.stageName}}/host/{{ agentStat.agent.hostName }}"
+                                           type="button" class="deployToolTip btn btn-xs {{ agentStat | agentButton }} host-btn"
+                                           title="{{ agentStat | agentTip }}">
+                                           {% if report.showMode == "complete" %}
+                                           <small>{{ agentStat.agent.hostName }}</small>
+                                           {% endif %}
+                                           <i class="fa fa-fw {{ agentStat | agentIcon }}"></i>
+                                        </a>
+                                    {% endfor %}
+                                {% endif %}
+                            </small>
+                        </td>
+                        <td class="col-lg-1 text-right">
+                            <small>
+                            <a href="hosts/show?stage=TODO" class="deployToolTip" data-toggle="tooltip"
+                               title="Total new hosts which are being deployed to for the first time">
+                                {{ report.firstTimeAgentStats|length }}
+                            </a>
+                            </small>
+                        </td>
+                    </tr>
+                </table>
+            </div>
+        {% endif %}
+
+        {% if report.provisioningHosts %}
+            <div class="panel panel-default table-responsive">
+                <table class="table table-condensed table-hover">
+                    <tr>
+                        <td class="col-lg-2">
+                            <span class="deployToolTip" data-toggle="tooltip"
+                               title="New hosts to this environment and currently being provisioned.">
+                            <small>Provisioning</small>
```

Comment:
Provisioned

Reply:
leaving the naming the same for now and will iterate to what you've suggested after user feedback